components/icons/CirclePlusIcon.tsx

```diff
@@ -0,0 +1,18 @@
+const CirclePlusIcon: React.FC = ({}) => (
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
```suggestion
      fillRule="evenodd"
      clipRule="evenodd"
```
react tends to get annoyed regarding these